core/src/test/scala/com/salesforce/op/stages/impl/insights/RecordInsightsLOCOTest.scala

```diff
@@ -96,7 +96,7 @@ class RecordInsightsLOCOTest extends FlatSpec with TestSparkContext with RecordI
     )
   )
   // scalastyle:on
-  Spec[RecordInsightsLOCO[_]] should "work with randomly generated features and binary logistic regression" in {
+  it ("should work with randomly generated features and binary logistic regression") {
```

Comment:
replace `it` with `Spec[RecordInsightsLOCO[_]]`

Reply:
Can't directly do that with FunSpec, what worked best was wrapping `describe(Spec[RecordInsightsLOCO[_]])`  around the tests.

What the updated tests look like:

<img width="1139" alt="Screen Shot 2019-10-07 at 10 32 14 AM" src="https://user-images.githubusercontent.com/16666978/66336009-121a8100-e8f1-11e9-901c-9657253f0bd0.png">